host/config/config.json

```diff
@@ -1,12 +1,24 @@
+
 {
-    "bind": "0.0.0.0:8080",
-    "cache": "/tmp/sgx",
-    "log_path": "./log/raiko",
-    "concurrency_limit": 10,
-    "max_log_days": 7,
-    "log_level": "info",
     "network": "taiko_a7",
+    "rpc": "https://rpc.hekla.taiko.xyz/",
+    "l1_rpc": "https://l1rpc.hekla.taiko.xyz/",
+    "beacon_rpc": "https://l1beacon.hekla.taiko.xyz",
+    "block_no": 1,
```

Comment:
Would remove these from the config because not really default values, better for raiko to just give an error instead if they are not set by the user I think.